go.sum

```diff
@@ -38,7 +38,6 @@ cloud.google.com/go/storage v1.0.0/go.mod h1:IhtSnM/ZTZV8YYJWCY8RULGVqBDmpoyjwiy
 cloud.google.com/go/storage v1.5.0/go.mod h1:tpKbwo567HUNpVclU5sGELwQWBDZ8gh0ZeosJ0Rtdos=
 cloud.google.com/go/storage v1.6.0/go.mod h1:N7U0C8pVQ/+NIKOBQyamJIeKQKkZ+mxpohlUTyfDhBk=
 cloud.google.com/go/storage v1.8.0/go.mod h1:Wv1Oy7z6Yz3DshWRJFhqM/UCfaWIRTdp0RXyy7KQOVs=
-cloud.google.com/go/storage v1.10.0 h1:STgFzyU5/8miMl0//zKh2aQeTyeaUH3WN9bSUiJ09bA=
```

Comment:
These changes seem quite unrelated. Could we drop them?

Reply:
weird I just run update-codegen.sh

Reply:
might be because you're using a newer Golang version or something.

Reply:
but check here: https://github.com/knative/operator/pull/566/files#diff-3295df7234525439d778f1b282d146a4f1ff6b415248aaac074e8042d9f42d63L41

This is auto-generated as well.